content/20.pharmaceuticals.md

```diff
@@ -493,19 +486,19 @@ However, the current crisis has underscored how different areas of biomedical re
 | SNG001 | B | Biologic, interferon | None | RCT | Mixed results: support from initial RCT but no effect found in WHO's Solidarity trial |
 
 Table: Summary table of candidate therapeutics examined in this manuscript.
-Grade is the rating given to each treatment by the Systematic Tracker of Off-label/Repurposed Medicines Grades (STORM) maintained by the Center for Cytokine Storm Treatment & Laboratory (CSTL) at the University of Pennsylvania [@url:https://cdcn.org/corona-data-viewer].
+"Grade" is the rating given to each treatment by the Systematic Tracker of Off-label/Repurposed Medicines Grades (STORM) maintained by the Center for Cytokine Storm Treatment & Laboratory (CSTL) at the University of Pennsylvania [@url:https://cdcn.org/corona-data-viewer].
 A grade of A indicates that a treatment is considered effective, B that all or most RCTs have shown positive results, C that RCT data are not yet available, and D that multiple RCTs have produced negative results.
 Treatments not in the STORM database are indicated as N/A.
 FDA status is also provided where available.
 The evidence available is based on the progression of the therapeutic through the pharmaceutical development pipeline, with randomized controlled trials (RCT) as the most informative source of evidence.
 The effectiveness is summarized based on the current available evidence; large trials such as RECOVERY and Solidarity are weighted heavily in this summary.
 This table was last updated on August 2, 2021.
-{#tbl:summary-table}
+{#tbl:summary-table} <!---In the rendered version, the caption shows up as "Table 1: Table 1:" Maybe it got fixed..-->
```

Comment:
This is an issue with the table and figure legends in the docx rendering-- I have to figure out where in the pipeline it's happening, but for now I've just been deleting them manually when I send out the docx. I must have missed this one!

Reply:
There are a few open issues at https://github.com/tomduck/pandoc-tablenos/issues (28, 31) that could be related.